src/test/resources/listoffiles.csv

```diff
@@ -0,0 +1,2 @@
+filename
```

Comment:
can we move these files into a subfolder called `casesGenerator\documents`

Reply:
Would like to keep test related static data files under resource folder. that's fine